pages/en/billing.mdx

```diff
@@ -2,75 +2,128 @@
 title: Billing on Subgraph Studio
 ---
 
-### Overview
+## Billing on Arbitrum
 
-For a quick demo of how billing works on the Subgraph Studio, check out the video below:
+Invoices are statements of payment amounts owed by a customer and are typically generated on a weekly basis in the system. You'll be required to pay fees based on the query fees you generate using your API keys. While The Graph protocol operates on Ethereum Mainnet, the billing contract lives on the [Arbitrum](https://arbitrum.io/) network to reduce transaction times and cost. The billing contract will allow you to:
```

Comment:
It's a little bit strange that after the title Billing on Arbitrum we have the first sentence "Invoices are statements..." . People usually know what invoices are. I would start with "While the graph protocol operates.." sentence, and update the "Invoices" statement. Like this:
"While The Graph protocol operates on Ethereum Mainnet, the billing contract lives on the [Arbitrum](https://arbitrum.io/) network to reduce transaction times and cost. You'll be required to pay fees based on the query fees you generate using your API keys. Invoices are typically generated on a weekly basis. Using the Billing, you'll be able to:
- Add and remove GRT...
- .....
-
"

cc @MichaelMacaulay

Reply:
Maybe "weekly basis" is bolded?

Reply:
Better yet, maybe a disclaimer message at the top (similar to how we did the other setions)

<img width="862" alt="Screenshot 2022-12-07 at 4 01 08 PM" src="https://user-images.githubusercontent.com/56690114/206294777-d00353f1-3359-43e7-87ca-2e327a5f73d7.png">